File: api/Newmark.PropertyManagement.Domain/Property.cs
using Newmark.PropertyManagement.Domain.Models;

namespace Newmark.PropertyManagement.Domain
{
    public class Property
    {
        public string PropertyId { get; set; }
        public string PropertyName { get; set; }
        public string[] Features { get; set; }
        public string[] Highlights { get; set; }
        public Transportation[] Transportation { get; set; }
        public Space[] Spaces { get; set; }
    }

    public class PropertyResponse
    {
        public Property[] Properties { get; set; }
    }
}

File: ui/src/types/property.ts
export interface RentRoll {
  month: string
  rent: number
}

export interface Space {
  spaceId: string
  spaceName: string
  rentRoll: RentRoll[]
}

export interface Property {
  propertyId: string
  propertyName: string
  features: string[]
  hightlights: string[]
  transportation: Transportation[]
  spaces: Space[]
}

export interface Transportation {
  type: string,
  line: string,
  distance: string,
  station: string
}

export interface PropertyResponse {
  properties: Property[]
}
